src/dotnet/14.1/Imports/Microsoft.Common.props/ImportBefore/Microsoft.NuGet.ImportBefore.props

```diff
@@ -0,0 +1,17 @@
+<!--
+***********************************************************************************************
+Microsoft.NuGet.ImportBefore.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <NuGetProps Condition="'$(NuGetProps)'==''">$(MSBuildExtensionsPath)\Microsoft.NuGet.props</NuGetProps>
```

Comment:
Where is this property set? MSBuild sets it automatically? Or does the user have to do something, like install VS?